### ISSUE TITLE
Implement the SpellBook

### ISSUE THREAD

**Issue by DeltaEvo** [aresrpg/aresrpg#17]
Depends on #16 #14

- Add the SpellBook Item that have a list of spells
- When the player shift + scoll on the spellbook change the active spell
- When the player click on the spellbook launch a spell

**Comment by ToufouMaster** [comment]
Hi can you explain me why it's blocked?

**Comment by Sceat** [comment]
depending on #16

**Comment by ToufouMaster** [comment]
i'm actually making both
#16 and #15

 #15 is already working and #16 need a bit more of adjustment but everything is working

**Comment by Sceat** [comment]
Nice, can you make a draft PR for us to follow ?

**Comment by Sceat** [comment]
Also that spellbook should be replaced by a fight oriented secondary hotbar, and the spells launched with numkeys

**Comment by DeltaEvo** [comment]
I think we should do both of them not replace the old one

**Comment by Sceat** [comment]
Superseded by #582 (using a new launch interface)
